crates/biome_analyze/src/rule.rs

```diff
@@ -227,6 +229,28 @@ impl RuleSourceKind {
     }
 }
 
+#[derive(Debug, Default, Clone)]
+pub enum LanguageKind {
+    #[default]
+    Js,
+    Json,
+    Css,
+}
```

Comment:
Is there any chance that we can avoid adding language metadata inside the generic crate?

Reply:
I tried a bit, but it seems difficult for now. Each 'registry' will be visited, and rules will be stored in a BTreeMap, but since the visitor consolidates each nursery rules into one 'nursery', it becomes impossible to know the target language of those rules.

Reply:
I see. Can we revert the change for now? I need some time to think about it

Reply:
Done👍